content/en/deployment/kubernetes/_index.md

```diff
@@ -3,8 +3,8 @@ title: "Kubernetes Deployment in Armory CD-as-a-Service"
 linktitle: "Kubernetes"
 weight: 10
 no_list: false
```

Comment:
`exclude_search: true`  on the _index.md pages means the section pages don’t show up in the site search results. Since we are including an Overview page in relevant section, I don't see the value of including section/_index.md in the search results. Thoughts?